ocr_tamil/craft_text_detector/file_utils.py

```diff
@@ -141,23 +158,27 @@ def crop_poly(image, poly):
     return cropped
 
 
-def export_detected_region(image, poly, rectify=True):
+def export_detected_region(image, poly, method="rectify"):
```

Comment:
Please change this default to crop  and you are good to g

Reply:
@JamesDConley

Reply:
Updated!